docs/patterns/observability/logs.md

```diff
@@ -16,14 +16,24 @@ To customize log forwarding to your output, follow these steps and then reinstal
 
 2. The default configuration uses CRI log format parsing. If you are using a kubernetes distribution that causes your logs to be formatted differently, you may need to add a new parser to the `[PARSER]` section and change the `parser` name in the `[INPUT]` section to one of the parsers defined [here](https://github.com/fluent/fluent-bit/blob/master/conf/parsers.conf).
 
-3. To view all logs irrespective of log level, you may remove the `[FILTER]` section. To change the log level being filtered on, you can update the "error" value below to "debug", "info", "warn", "fatal", "panic" or "trace":
+3. The filters are currently customized to match "error" level logs and multiple filters have been used to cover differences in log formatting on various k8s distros. To view all logs irrespective of log level, you may remove the `[FILTER]` sections. 
+
+    To change the log level being filtered on, you can update the "error" values below to "debug", "info", "warn", "fatal", "panic" or "trace":
```

Comment:
are you referring to the error value in the `Condition` attribute? What do you think about templating this so that the user can update a value in the values.yaml file and reference that value in the `fluentbit-configmap.yaml` to make configuration of the error level easier?